q2_longitudinal/assets/index.html

```diff
@@ -2,6 +2,25 @@
 
 {% block content %}
 
+{% if errors %}
+<div class="panel-group" id="accordion" role="tablist" aria-multiselectable="true">
```

Comment:
These ids (and the others) will probably collide with anything else that uses this in the future. Maybe make it `warnings-accordion`?

Reply:
(id attributes, e.g. `#something` are expected to be unique on the page)